src/style/theme.scss

```
@@ -22,6 +22,7 @@ $themes: (
     tertiary-button-background: $tertiary-button-background-light,
     tertiary-button-background-variant-1: $tertiary-button-background-light-variant-1,
     card: $card-light,
+    card-background: $card-background,
```

If this color is related to the disabled status, I will have added it more explicitly

```suggestion
    card-background-disabled: $card-background,
```

---

I had a doubt about it yesterday